holidays/countries/singapore.py

```diff
@@ -58,6 +58,7 @@ def __init__(
         subdiv: Optional[str] = None,
         prov: Optional[str] = None,
         state: Optional[str] = None,
+        language: Optional[str] = None,
```

Comment:
This class (as well as MY) has its own __init__() signature. The new param is added to keep it consistent. Otherwise SG wouldn't have `language` available.

Reply:
Oh, yeah! Now I remember, I did it early on in the project in order to surface the docstring in IDEs. This was before we standardized on using `holidays.country_holidays()`, which provides docstring no matter what country you call.

I will edit it away in a separate, forthcoming, PR.

Thanks and apologies.